docs/resources/privileged_access_group_assignment_schedule_request.md

```diff
@@ -0,0 +1,70 @@
+---
+subcategory: "Identity Governance"
+---
+
+# Resource: azuread_privileged_access_group_assignment_schedule_request
```

Comment:
This resource may have similar problems as https://github.com/hashicorp/terraform-provider-azuread/issues/1306.

So, you should do follows:

1. remove the `azuread_privileged_access_group_assignment_schedule_request` resource entirely
2. replace it with the `azuread_privileged_access_group_assignment_schedule` resource
3. have the provider logic manage the steps of creating a schedule request and linking the terraform resource to the id of the created schedule instead of the schedule request so we can actually see if the schedule needs to be recreated

See also: https://learn.microsoft.com/en-us/graph/api/privilegedaccessgroupassignmentschedule-get

Reply:
😢

There is a problem on the other side of the coin which you haven't run into. If a request is future-dated, then the schedule is not created until that future date (plus or minus a few minutes 🤯). So if we want to support future-dated schedules, we need to work out how to handle both halves.

Reply:
I've moved to the `azuread_privileged_access_group_assignment_schedule` resource are recommended. I've catered (I think) for both future start dates, and expiry of a time-bound schedule